_docs/howto/configure/gsuite-group-collection.md

```diff
@@ -66,11 +66,11 @@ You must have the **super admin** role in admin.google.com to complete these ste
 After you create a service account above, you may need to edit the following variables 
 in your `forseti_conf.yaml`.
 
-- `groups-domain-super-admin-email`: Use of the Admin API requires delegation
+- `domain_super_admin_email`: Use of the Admin API requires delegation
```

Comment:
Just a comment, and not a request for change.  It would be so much simpler to rename these as `gsuite_super_admin_email` and `gsuite_service_account_key_file`.